pyemu/utils/geostats.py

```diff
@@ -1000,8 +1001,10 @@ def calc_factors_grid(
 
                 dfs.append(df)
                 if var_filename is not None:
+                    #a = np.array([float(str(i)) for i in df.err_var],dtype=np.float).reshape(x.shape)
+                    #a = df.err_var.values.reshape(x.shape)
                     a = df.err_var.values.reshape(x.shape)
-                    na_idx = np.isfinite(a)
+                    na_idx = ~np.isnan(a)
```

Comment:
are you happy that `a` with never contain `inf` or `-inf`?

Reply:
Ah yeah - should go back to infinite.  That was a leftover from the dig